src/core.js

```diff
@@ -441,7 +441,6 @@ $.extend( $.validator, {
 			if ( errors ) {
 				// add items to error list and map
 				$.extend( this.errorMap, errors );
-				this.errorList = [];
```

Comment:
Did test this change against the fiddle provided by @codezzly if it fixes our problem?